packages/editor/src/components/widgets/fieldset/Fieldset.css

```diff
@@ -25,10 +25,10 @@
   gap: var(--size-2);
 }
 .fieldset-input.fieldset-error {
-  --input-border: 1px solid var(--error-color);
+  --basic-border: 1px solid var(--error-color);
```

Comment:
be careful here. 
Everything inside this which uses this variable will have a red or yellow border